sklearn/metrics/cluster/supervised.py

```diff
@@ -625,8 +625,8 @@ def mutual_info_score(labels_true, labels_pred=None, contingency=None):
                                              sparse=True)
     else:
         warnings.warn("Contingency is deprecated in 0.19 and will be removed" +
-                      "in 0.21 to use contingency matrix pass only contingency"
-                      + " matrix to scoring function", DeprecationWarning)
+                      "in 0.21 to use contingency matrix pass only contingen" +
+                      "cy matrix to scoring function", DeprecationWarning)
```

Comment:
I'll have to into this. Maybe tomorrow?
And for my comment, although the flake8 tests pass, it's not good practice to break words like this. Please make the change for brevity.